tests/integration/commands/dev/dev-forms-and-redirects.test.ts

```diff
@@ -497,11 +513,12 @@ describe.concurrent('commands/dev-forms-and-redirects', () => {
             async (server) => {
               const [response1, response2] = await Promise.all([
                 fetch(`${server.url}/foo`).then((res) => res.text()),
-                fetch(`http://localhost:${userServerPort}`).then((res) => res.text()),
+                fetch(`http://localhost:${userServerPort.toString()}`).then((res) => res.text()),
               ])
-              await t.expect(response1).toEqual('<html><h1>foo')
-              await t.expect(response2).toEqual('Hello world')
+              t.expect(response1).toEqual('<html><h1>foo')
+              t.expect(response2).toEqual('Hello world')
             },
+            // @ts-expect-error TS(2345) FIXME: Argument of type '{ message: RegExp; }' is not ass... Remove this comment to see the full error message
             { message: /Error: Something went wrong/ },
```

Comment:
I traced this and it just happens to be "working" because it's truthy
```suggestion
            true,
```

Reply:
Fixed